packages/dapp/src/components/FeatureCards/FeatureCard.tsx

```diff
@@ -0,0 +1,133 @@
+import CloseIcon from '@mui/icons-material/Close';
+import { IconButton, Link, Slide, useTheme } from '@mui/material';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import { useEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSettingsStore } from '../../stores';
+import { Card, CardImage } from './FeatureCard.style';
+
+export const FeatureCard = ({ data, isSuccess, assets }) => {
+  const [open, setOpen] = useState(true);
+  const { t: translate } = useTranslation();
+  const [lightImageUrl, setLightImageUrl] = useState('');
+  const [darkImageUrl, setDarkImageUrl] = useState('');
+  const [onDisableFeatureCard] = useSettingsStore((state) => [
+    state.onDisableFeatureCard,
+  ]);
+  const i18Path = 'featureCard.';
+  const theme = useTheme();
+
+  useEffect(() => {
+    data?.fields?.displayConditions &&
+      data?.fields?.displayConditions[0]?.showOnce &&
+      onDisableFeatureCard(data?.fields?.displayConditions[0]?.id);
+  }, [data?.fields?.displayConditions, onDisableFeatureCard]);
+
+  const imageUrl = useMemo(() => {
+    return assets.filter((el) => {
+      return theme.palette.mode === 'dark'
+        ? el?.sys?.id === data?.fields?.imageDarkMode?.sys?.id
+        : el?.sys?.id === data?.fields?.imageLightMode?.sys?.id;
+    })[0].fields?.file?.url;
+  }, [
+    theme.palette.mode,
+    assets,
+    data?.fields?.imageDarkMode?.sys?.id,
+    data?.fields?.imageLightMode?.sys?.id,
+  ]);
+
+  return (
+    <Slide
+      direction="up"
+      in={open}
+      unmountOnExit
+      appear={false}
+      timeout={150}
+      easing={'cubic-bezier(0.32, 0, 0.67, 0)'}
+    >
+      <Card gradient={data?.gradientColor}>
+        <CardContent sx={{ padding: theme.spacing(6), position: 'relative' }}>
+          <IconButton
+            disableRipple={true}
+            sx={{
+              position: 'absolute',
+              right: theme.spacing(2),
+              top: theme.spacing(2),
+            }}
+            onClick={() => {
+              setOpen(false);
+              !data?.fields?.displayConditions[0].hasOwnProperty('showOnce') &&
+                !!data?.fields?.displayConditions[0]?.id &&
+                onDisableFeatureCard(data?.fields?.displayConditions[0]?.id);
+            }}
+          >
+            <CloseIcon
+              sx={{
+                width: '24px',
+                height: '24px',
+                color:
+                  theme.palette.mode === 'dark'
+                    ? theme.palette.white.main
+                    : theme.palette.black.main,
+              }}
+            />
+          </IconButton>
+
+          {!isSuccess ? (
+            'loading'
```

Comment:
Do we show the loading state as just `loading` to he user? 👀

Reply:
Now tested it!
The wrapper "FeauterCards.tsx" is fetching all data from Contentful and loops through it once successful.
When rendering each FeatureCard, it cannot be NOT succesful anymore - so that 'loading' is useless overall and a bad solution anyways!

Just for the heck of it, I played with the idea to add some loading effects but that should probably be done seperately and improved in a different ticket - as there might be discussions on how the Feature-Cards should be populated.
Right now, they just pop out of nowhere and sure that can be improved